### PR TITLE
namespace known issue--in review

### PR DESCRIPTION
**Pull request by bcpswope**
https://github.com/open-cluster-management/backlog/issues/2311

### CODE REVIEW

File: release_notes/known_issues.md [hunk 1]
```diff
@@ -174,5 +174,10 @@ For more information, see [Certificate policy controller](../security/cert_polic
...
 
+For technical preview, documentation links were removed from the console, but a few might still be exposed. Any links to the documentation for preview are temporarily not updated with the correct links.
+
+## Application channels require unique namespaces
+<!--1.0.0:2311-->
 
+Creating a channel in the same namespace can cause errors with the hub cluster. For instance, a namespace that is created for both the channel and the hub cluster, such as namepace `charts-v1`, will cause errors. Ensure that you create your channel in a unique namespace. All channels need an individual namespace, except GitHub channels, which can share a namespace with another channel. See the process for Managing channles(../manage_applications/managing_channels.md) for more information.
```
Comment (jnpacker) [inline release_notes/known_issues.md:182]: @rokej please confirm, but Github support for shared namespace will be a 2.0GA feature.

For the first two sentences.
Creating a channel in a namespace that already contains a channel resource will cause errors with the hub cluster. For instance,  there is a `charts-v1` channel that is used by the installer. You should not create any additional channels in this namespace. We recommend creating a new namespace for each channel.

Comment (rokej) [reply release_notes/known_issues.md:182]: You can have multiple GitHub type channels in the same namespace in tech preview release. For any other type channel, no.

Comment (jnpacker) [reply release_notes/known_issues.md:182]: Maybe a new paragraph after that says In Technical Preview, you can place multiple channels of type: git ONLY in the same namespace.

Comment (bcpswope) [reply release_notes/known_issues.md:182]: I have this line in the doc itself but I will add it here, sure.

**Comment by bcpswope** [comment]
@jnpacker please see:

Creating a channel in the same namespace can cause errors with the hub cluster. 

For instance, namespace `charts-v1` is used by the installer, so do not create any additional channels in `charts-v1`. Ensure that you create your channel in a unique namespace. 

For technical preview, all channels need an individual namespace, except GitHub channels, which can share a namespace with another channel. See the process for [Managing channels](../manage_applications/managing_channels.md) for more information.

**Pull request merged**